lib/submit-analytics.js

```diff
@@ -23,6 +23,77 @@
 
 'use strict';
 
+let utils = require('./utils');
+
+const require_method = require;
+function node_require(module) {
+    return require_method(module);
+}
+
+class webhookProdDetails {
+    /**
+     * Path and credentials required to submit analytics through the webhook (production mode).
+     */
+    constructor() {
+        this.addressPrefix =
+                "https://webhooks.mongodb-realm.com/api/client/v2.0/app/realmsdkmetrics-zmhtm/service/metric_webhook/incoming_webhook/metric?data=";
+        this.addressSuffix = "&ip=1";    }
+
+    /**
+     * Constructs the full URL that will submit analytics to the webhook.
+     * @param  {Object} payload Information that will be submitted through the webhook.
+     * @returns {string} Complete analytics submission URL
+     */
+    buildRequest(payload) {
+        const request = this.addressPrefix + Buffer.from(JSON.stringify(payload.webHook), 'utf8').toString('base64') + this.addressSuffix;
```

Comment:
I would probably break this into local variables with appropriate names to increase readability.
```suggestion
        const jsonPayload = JSON.stringify(payload.webHook);
        const base64Payload = Buffer.from(jsonData, 'utf8').toString('base64');
        const request = this.addressPrefix + base64Payload + this.addressSuffix;
```